src/contracts/core/AVSDirectory.sol

```diff
@@ -282,7 +287,7 @@ contract AVSDirectory is
             });
         }
 
-        uint32 effectTimestamp = uint32(block.timestamp) + ALLOCATION_DELAY;
+        uint32 effectTimestamp = uint32(block.timestamp) + DEALLOCATION_DELAY;
```

Comment:
needs to be custom allocation delay?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/691/commits/db66da58945da71f7f53940f6dbf656a7d25d61e